### PR TITLE
Added ADMIN ROUTES

### PR DESCRIPTION
**Pull request by methebao**

### CODE REVIEW

**Comment by methebao** [comment]
ready

File: src/app/layout/index.js [hunk 1]
```diff
@@ -1,35 +1,72 @@
...
+class Layout extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      routeLayout: ROUTE_LAYOUT.USER
+    };
+  }
+  componentDidMount() {
+    const { history } = this.props;
+    debugger;
```
Comment (chien-pv) [inline src/app/layout/index.js]: xóa debugger

**Comment by chien-pv** [comment]
commented

File: src/app/layout/index.js (hunk 1 above)
Comment (methebao) [reply src/app/layout/index.js]: oke a :D

**Comment by methebao** [comment]
ready

**Pull request merged**